bazel/cc_toolchains/clang_configuration.bzl

```diff
@@ -129,21 +129,22 @@ def _configure_clang_toolchain_impl(repository_ctx):
         sysroot_dir,
     )
 
-    # Ensure the detected clang path has some LLVM tools in it, in attempt to
-    # warn the user if they don't have these necessary LLVM tools in their PATH.
-    # This check isn't exhaustive, only a best attempt at warning the developer.
-    llvm_check_result = repository_ctx.execute([
-        clang.dirname.get_child("llvm-ar"),
-        "--version"
-    ])
-    if llvm_check_result.return_code != 0:
-        fail("`llvm-ar` not found beside clang: is LLVM in PATH?")
+    # We assume that the LLVM binutils live adjacent to llvm-ar.
+    # First assume llvm-ar is adjacent to clang, so that if found,
```

Comment:
```suggestion
    # We expect that the LLVM binutils live adjacent to llvm-ar.
    # First look for llvm-ar adjacent to clang, so that if found,
```

Reply:
Minor wording edit, I'm just slipping it in.